frontend/js/utils/api.js

```diff
@@ -1,13 +1,36 @@
 import axios from 'axios';
 
+import Urls from './urls';
+
 const baseUrl = window.location.host;
 
-const getFromApi = (apiUrls) => {
-  const url = `http://${baseUrl}${apiUrls}`;
+const createTeamUrl = (id) => {
```

Comment:
This function is inconsistent with the rest of the functions, please change this to follow the pattern of the others